content/espanol/python-guess-number/answer-key.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Python: Adivina el Número - Respuestas"
+date: 2020-03-27T20:24:33-07:00
+weight: 15
+draft: false
+hidden: true
+---
+
+## Jugando con randit
+
+```python
+import random
+
+number = random.randint(1,100)
+print("tu número generado aleatoriamente es", number)
+```
+
+## Crear el juego 'Adivina el número'
+
+```python
```

Comment:
One new interesting caveat is that currently the translation refers to trinket code snippets in English... someone will need to manually create new Spanish code snippets in Trinket and refer to those in the workshop

Reply:
ya unfortunately it still needs manual intervention to update those code snippet links